src/site-packages/openbmp/mrt2bmp/HelperClasses.py

```diff
@@ -0,0 +1,36 @@
+import shutil
+import os
+
+# Message class to put and deque from intermediate message queue.
+class Q_Message(object):
+
+    # Type of message in the queue. Message types are bmp message and reconnect message.
+    RECONNECT_MESSAGE = 0
+    BMP_MESSAGE = 1
+
+    def __init__(self, msg_type, msg=""):
+        self.msg_type = msg_type
+        self.msg = msg
+
+    def __str__(self):
+        if self.msg_type == self.RECONNECT_MESSAGE:
+            return "{ RECONNECT_MESSAGE }"
+
+        elif self.msg_type == self.BMP_MESSAGE:
+            return "{ BMP_MESSAGE, %s}" % self.msg
+
+    def getMsgType(self):
+        return self.msg_type
+
+    def getMessage(self):
+        return self.msg
+
+
+def moveFileToTempDirectory(src_file_path, dst_dir_path):
+
+    # Checks if dst directory exists. If not, then creates the directory structure.
+    if not os.path.isdir(dst_dir_path):
+        os.makedirs(dst_dir_path)
+
+    shutil.move(src_file_path, dst_dir_path)
```

Comment:
Need to add try, except shutil.Error...  This is failing at times and causing the program to hang.